api-docs/apiLive.htm

```diff
@@ -30709,7 +30717,70 @@ <h2 class="flybar-button">Template</h2>
 				</div>
 			</div>
 
+			<!-- start of password validation policy api docs -->
+			<a id="password_validation_policy" name="passwordValidationPolicy" class="old-syle-anchor">&nbsp;</a>
```

Comment:
In an effort to consolidate Password related preferences behind a single API (https://mifosforge.jira.com/browse/MIFOSX-1928), this API has been renamed to "passwordpreferences"